docs/sources/developers/plugins/add-authentication-for-data-source-plugins.md

```diff
@@ -154,3 +154,30 @@ Grafana automatically renews the token when it expires.
   }
 ]
 ```
+
+### Add headers to OAuth request
```

Comment:
I would add the description that optional headers can be set in the "Enable token authentication" section.